### PR TITLE
Feat/sok ddagen

### PR DESCRIPTION
**Pull request by rikaznismi**

### CODE REVIEW

**Comment by rikaznismi** [comment]
I will clean the code a bit in the meanwhile

File: src/pages/sok.tsx [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+      </h1>
+
+      <SingleEvent
+        bgColor="bg-[#E2B7C9]"
+        borderColor="border-[#E2B7C9]"
+        toReverse={false}
+        textColor="text-[#E2B7C9]"
+        image="/img/groupPictures/prTeam.jpg"
+        eventInfo={[
+          t.sok.prGroup.header,
```
Comment (ernum) [inline src/pages/sok.tsx:151]: The headers should probably be centered?

image

File: src/pages/sok.tsx [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+        {/*MODAL*/}
+        {modalState && (
+          <div
+            className="fixed inset-0 flex items-center justify-center bg-black bg-opacity-60 backdrop-blur-lg"
+            ref={modalRef}
+            onClick={handleOverlayClick}
+          >
+            <div
+              className={`bg-white bg-opacity-70 w-[500px] max-h-[80vh] overflow-y-auto pb-5 flex flex-col rounded-3xl`}
+              style={{ scrollbarWidth: "none", msOverflowStyle: "none" }}
```
Comment (ernum) [inline src/pages/sok.tsx:72]: I can still see the scrollbar. Do we want it?
image

File: src/pages/sok.tsx [hunk 3]
```diff
@@ -0,0 +1,213 @@
+import { useLocale } from "@/locales";
```
Comment (ernum) [inline src/pages/sok.tsx:1]: We can still scroll outside the block:

image

File: src/pages/sok.tsx [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+      <div className="flex flex-row items-center justify-center mt-[50px]">
+        <p className="text-white w-[300px] sm:w-[600px] text-center">
+          {t.sok.description}
+        </p>
+      </div>
+      <a href="https://forms.gle/VyigeGiQuEXgE9eS8" target="_blank">
+      <div className="flex flex-row items-center justify-center">
+        <button className="w-[200px] bg-cerise rounded-full h-[60px] mt-[50px]">
+          <p className="text-white text-center"> {t.sok.search}</p>
+        </button>
```
Comment (ernum) [inline src/pages/sok.tsx:136]: Text padding could be less in button IMO but its a preference
image

File: src/pages/sok.tsx [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+              </div>
+            </div>
+          </div>
+        )}
+      </div>
+    </div>
+  );
+}
+
+export default function Students() {
```
Comment (ernum) [inline src/pages/sok.tsx:118]: This the page function name should be changed to "Sok" or something similar

File: src/pages/sok.tsx [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+  const t = useLocale();
+
+  return (
+    <div className="pt-[200px] pb-[300px]">
```
Comment (ernum) [inline src/pages/sok.tsx:122]: The space between the last button and the footer is a bit too large.

image

**Review by ernum — CHANGES_REQUESTED** [review]

File: src/pages/sok.tsx (hunk 6 above)
Comment (rikaznismi) [reply src/pages/sok.tsx]: Fixed

File: src/pages/sok.tsx (hunk 5 above)
Comment (rikaznismi) [reply src/pages/sok.tsx]: Fixed

File: src/pages/sok.tsx (hunk 4 above)
Comment (rikaznismi) [reply src/pages/sok.tsx]: I made button smaller overall, which gives a similar effect i believe

File: src/pages/sok.tsx (hunk 3 above)
Comment (rikaznismi) [reply src/pages/sok.tsx:1]: Scrolling is now disabled when a team is clicked on, thereby not making it posible to scroll outside the block

**Review by ernum — APPROVED** [review]

**Pull request merged**